src/scss/tom-select.bootstrap4.scss

```diff
@@ -1,19 +1,18 @@
 /**
- * Tom Select bootstrap 4
+ * Tom Select Bootstrap 4
  */
 
-//Import Boostrap 4 functions and variables
+// Import Bootstrap 4 functions and variables
 
 $state-valid: map-get($form-validation-states,'valid') !default;
 $state-invalid: map-get($form-validation-states,'invalid') !default;
-
 $enable-shadows: true !default;
 $select-font-family: inherit !default;
 $select-font-size: inherit !default;
-$select-line-height: $input-btn-line-height !default; //formerly line-height-computed
+$select-line-height: $input-btn-line-height !default; // formerly line-height-computed
 
-$select-color-text: gray("800") !default; //$gray-800
-$select-color-highlight: rgba(255,237,40,0.4) !default;
+$select-color-text: gray("800") !default; // $gray-800
+$select-color-highlight: rgb(255 237 40 / 40%) !default;
```

Comment:
Double check that these modern colors work as expected, if not silence the style lint rule and convert back to classic.